src/omnicore/rpctxobject.cpp

```diff
@@ -497,12 +497,15 @@ void populateRPCTypeActivation(CMPTransaction& omniObj, Object& txobj)
     txobj.push_back(Pair("minimumversion", (uint64_t) omniObj.getMinClientVersion()));
 }
 
-void populateRPCExtendedTypeSendToOwners(const uint256 txid, std::string extendedDetailsFilter, Object& txobj)
+void populateRPCExtendedTypeSendToOwners(const uint256 txid, std::string extendedDetailsFilter, Object& txobj, uint16_t version)
 {
     Array receiveArray;
     uint64_t tmpAmount = 0, stoFee = 0;
     LOCK(cs_tally);
     s_stolistdb->getRecipients(txid, extendedDetailsFilter, &receiveArray, &tmpAmount, &stoFee);
+    if (version > MP_TX_PKT_V0) {
+        stoFee = stoFee * 100; // just awful, but until we start storing the fee somewhere it's a workaround
```

Comment:
Let's make it at least a constant somewhere, similar to the old fee in omnicore.h. :)